samwise.js

```diff
@@ -1,61 +1,63 @@
+const objectKeys = Object.keys(object);
```

Comment:
SCRIPT-Consider using more descriptive naming for this boolean variable to clarify its purpose.